config/software/omnibus-toolchain.rb

```diff
@@ -28,6 +28,9 @@
 # git!
 dependency "git"
 
+# fakeroot for EL based things
+dependency "fakeroot" if el?
```

Comment:
I don't know of any, I was following the example of the omnibus cookbook. I thought it was only used during RPM creation? I'm fine with removing this condition if we want to go that direction.